### DISCUSSION TITLE
Council Wards

### DISCUSSION THREAD

**Discussion by MrMikeRose** [digital-land/data-standards-backlog#74] [Planning consideration]
Chat here about council wards!

**Comment by colmjude** [comment]
### Existing data

Ward data is published in 2 places. By [ONS](https://geoportal.statistics.gov.uk/documents/4d9aa61b06a64ec1b7cc4aecbcfbb7a9/about) (the authoritative source) and by OS as part of their [boundary-line product](https://osdatahub.os.uk/downloads/open/BoundaryLine).

**Reply by colmjude** [reply]
This is the [latest dataset that has the geometries](https://geoportal.statistics.gov.uk/datasets/69b2084e13284ea59be314cb28edc41c_0/explore).

It has 8,396 records

A typical record looks like this

| field | value |
| -- | -- |
| FID | 5187 |
| WD24CD | E05014290 |
| WD24NM | Leyburn & Middleham |
| WD24NMW | |
| BNG_E | 412041 |
| BNG_N | 485644 |
| LONG | -1.816630 |
| LAT | 54.266319 |
| Shape__Area | 224122827.768219 |
| Shape__Length | 103117.081588 |

**Reply by colmjude** [reply]
We've created a [ward schema](https://github.com/digital-land/specification/blob/main/content/dataset/ward.md?plain=1)

Mappings from the available data are

| field | mapped from | default value |
| --- | --- | --- |
| reference | WD24CD | |
| name | WD24NM | |
| organisation | | government-organisation:D303 |
| start-date | | 2024-07-01 |

**Reply by colmjude** [reply]
An issue with the available dataset is that it is a snapshot. For example, we are looking to collect the lastest wards which are for july 2024. When ONS publish new or updated wards it will likely be a new dataset for us to collect.

Collecting from multiple endpoints is fine. However, the fields we are interested in might change. In the current dataset the fields we care about are `WD24CD` and `WD24NM`, but if we looking in a previous publication ([Wards December 2023](https://geoportal.statistics.gov.uk/datasets/ons::wards-december-2023-boundaries-uk-bfc-2/about)) the fields are `WD23CD` and `WD23NM`

**Reply by trodaway** [reply]
Those fields are named as `<abbreviation for geometry type - i.e. Ward = WD><2 digit year><abbreviation for field type - i.e. Code = CD and Name = NM>`. As such, these fields **will** change each time to ingest the data.

**Comment by trodaway** [comment]
Having seen a comment on Slack about Wards, I just had a nosey at the dataset. It's somewhat concerning that you're using the annually updated ONS dataset, opposed to the more frequently updated OS dataset. Long-term, it'd be good for you to use the OS NGD, so it's updated daily even. 

At the moment, you have wards which are two months out of date for our authority and likely others too. Thankfully we are not using this dataset, but it could be misleading to members of the public and impact systems (e.g. Plan X) that may in the future integrate with this dataset.
